### PR TITLE
Add SURFRAD page

### LINKED ISSUE #242: Add page on SURFRAD

### PR DESCRIPTION
**Pull request by IoannisSifnaios**
closes #242 

New page [here](https://solarstations--247.org.readthedocs.build/en/247/station_network_surfrad.html)

### CODE REVIEW

**Comment by IoannisSifnaios** [comment]
Note that the [SURFRAD website](https://gml.noaa.gov/grad/surfrad/sitepage.html) only mentions 8 stations while we have 10 in the list

**Comment by AdamRJensen** [comment]
> Note that the [SURFRAD website](https://gml.noaa.gov/grad/surfrad/sitepage.html) only mentions 8 stations while we have 10 in the list

Two of the stations are retired/inactive. This info is note in the Time period column, so I think this is ok.

File: docs/_toc.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ parts:
       title: MIDC
     - file: station_network_solrad
       title: SOLRAD
+    - file: station_network_surfrad
+      title: SURFRAD
     - file: station_network_srml
       title: SRML
     - file: station_network_esmap
```
Comment (AdamRJensen) [inline docs/_toc.yml:28]:
```suggestion
    - file: station_network_srml
      title: SRML
    - file: station_network_surfrad
      title: SURFRAD
    - file: station_network_esmap
```
List networks in alphabetical order.

File: docs/station_network_surfrad.ipynb [hunk 2]
```diff
@@ -0,0 +1,860 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SURFRAD\n",
+    "\n",
+    "The [SURFRAD](https://gml.noaa.gov/grad/surfrad/sitepage.html) network was established in 1993 and is operated by the [National Oceanic and Atmospheric Administration (NOAA)](https://www.noaa.gov/).\n",
```
Comment (AdamRJensen) [inline docs/station_network_surfrad.ipynb:9]:
```suggestion
    "The [SURFRAD](https://gml.noaa.gov/grad/surfrad/sitepage.html) network was established in 1993 and is operated by the [National Oceanic and Atmospheric Administration (NOAA)](https://www.noaa.gov/). The network features six active and two inactive stations in the contiguous United States. The stations are all Tier 1 stations and measurements are generally of a very high quality due to rigorous maintenance procedures and frequent inspection of data.\n",
```

**Comment by AdamRJensen** [comment]
Thanks @IoannisSifnaios 🥳

**Pull request merged**